var/spack/repos/builtin/packages/py-pyogrio/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPyogrio(PythonPackage):
+    """Vectorized spatial vector file format I/O using GDAL/OGR"""
+
+    homepage = "https://pypi.org/project/pyogrio"
+    pypi = "pyogrio/pyogrio-0.9.0.tar.gz"
+    git = "https://github.com/geopandas/pyogrio.git"
+
+    maintainers("climbfuji")
+
+    license("MIT", checked_by="climbfuji")
+
+    version("0.9.0", sha256="6a6fa2e8cf95b3d4a7c0fac48bce6e5037579e28d3eb33b53349d6e11f15e5a8")
+
+    depends_on("python@3.8:", type=("build", "run"))
+    depends_on("py-cython@0.29:", type="build")
+    depends_on("py-versioneer@0.28 +toml", type="build")
+    # this is an implicit dependency already listed in py-versioneer, not needed
+    # depends_on("py-tomli", when="^python@:3.10", type="build")
```

Comment:
(question)

* If this has any direct calls to `py-tomli` API, then an explicit dependency makes sense (although it makes sense why it wouldn't be strictly needed if `py-versioneer` is already depending on it
* Otherwise, it makes sense to drop the dependency here and let `py-versioneer` handle it

Does this not clearly fall into one of those two categories? Or does it and you think there is another reason to list this here?